aviary/interface/methods_for_level2.py

```diff
@@ -2361,16 +2361,30 @@ def _add_flops_landing_systems(self):
                                'traj.climb.initial_states:mass')
             self.model.connect(Mission.Takeoff.GROUND_DISTANCE,
                                'traj.climb.initial_states:range')
-            self.model.connect(Mission.Takeoff.FINAL_VELOCITY,
-                               'traj.climb.initial_states:velocity')
-            self.model.connect(Mission.Takeoff.FINAL_ALTITUDE,
-                               'traj.climb.initial_states:altitude')
+            if self.mission_method == "FLOPS":
+                self.model.connect(Mission.Takeoff.FINAL_VELOCITY,
+                                   'traj.climb.initial_states:velocity')
+                self.model.connect(Mission.Takeoff.FINAL_ALTITUDE,
+                                   'traj.climb.initial_states:altitude')
+            else:
+                pass
+                # TODO: connect this correctly
+                # mass is the most important to connect but these others should
+                # be connected as well
+                # self.model.connect(Mission.Takeoff.FINAL_VELOCITY,
+                #                 'traj.climb.initial_states:mach')
+                # self.model.connect(Mission.Takeoff.FINAL_ALTITUDE,
+                #                 'traj.climb.controls:altitude')
```

Comment:
Looks like for this case we are losing those initial connections, why did we do that?

Reply:
These connections remain for the FLOPS based mission. I've added commented out code here for the simple mission to show that we need to connect it later.

We cannot do these connections now because the simplified takeoff outputs velocity whereas the simple mission deals with Mach. We cannot do a direct connection but could do a constraint. Instead, I will make the simplified takeoff output Mach as well and connect that directly in a follow-on PR.

We do not lose any capability the changes in this PR; this is more of a note for the future.

Reply:
Understood, thanks for the explanation!